modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/scopes/RoutersExecutionTestCase.java

```diff
@@ -141,6 +143,12 @@ public void twoRoutesRouterNone() throws Exception {
   }
 
   @Test
+  public void stereotypedRoutes() throws Exception {
```

Comment:
this tests the happy path. There should be a test which verifies that the thing fails if you put an MP which doesn't fit the stereotype

Reply:
that is why I created this test `ScopeStereotypeValidationTestCase`